contracts/voting_escrow/tests/integration.rs

```diff
@@ -470,6 +470,22 @@ fn check_queries() {
     if (coeff - 1.02884f32).abs() > 1e-5 {
         assert_eq!(coeff, 1.02884f32)
     }
+
+    let total_vp_at_period = helper
+        .query_total_vp_at_period(router_ref, cur_period)
+        .unwrap();
+    let total_vp_at_ts = helper
+        .query_total_vp_at(router_ref, router_ref.block_info().time.seconds())
+        .unwrap();
+    assert_eq!(total_vp_at_period, total_vp_at_ts);
+
+    let user_vp_at_period = helper
+        .query_user_vp_at(router_ref, "user", cur_period)
```

Comment:
we need to pass the timestamp, not the period.

Reply:
Sure. Thank you